src/org/gk/persistence/XMLFileAdaptor.java

```diff
@@ -1076,10 +1100,10 @@ public void removeFromClassMap(GKInstance instance) {
     
     /**
      * Remove the delete record for a list of DB_IDs.
+     * 
      * @param dbIDs a list of DB_IDs.
-     * @throws IOException
      */
-    public void clearDeleteRecord(List<Long> dbIDs) throws IOException {
+    public void clearDeleteRecord(List<Long> dbIDs) {
```

Comment:
Why remove the `throws` declaration?

Reply:
(There are a number of instances of this, but I don't want to comment on each and every one....)

Reply:
It was removed since there was no place within the method body that throws an IOException.  I checked visually and it was confirmed with Eclipse.